bindings/go/descriptor/v2/descriptor.go

```diff
@@ -258,10 +262,28 @@ type SignatureInfo struct {
 // See https://github.com/open-component-model/ocm-spec/blob/main/doc/01-model/03-elements-sub.md#labels
 // +k8s:deepcopy-gen=true
 type Label struct {
-	Name  string `json:"name"`
-	Value string `json:"value"`
+	// Name is the unique name of the label.
+	Name string `json:"name"`
+	// Value is the json/yaml data of the label
+	Value json.RawMessage `json:"value"`
 	// Signing describes whether the label should be included into the signature
 	Signing bool `json:"signing,omitempty"`
+	// Version is the optional specification version of the attribute value
+	Version string `json:"version,omitempty"`
```

Comment:
what is the `Version` for? I don't see any usage or tests for it.

Reply:
The spec even says

> The specification version for the label content. If no version is specified, implicitly the version v1 is assumed.

So, we must set the version to `v1` if empty.

Reply:
no, we must not. we must assume v1, but it must not be serialized